build/scripts/Benchmarking.fsx

```diff
@@ -10,26 +16,131 @@ open System.Linq
 open System.Diagnostics
 open Paths
 
+open FSharp.Data
+
+open Nest
+open Elasticsearch.Net
+open Newtonsoft.Json
+open Git.Branches
+
 module Benchmarker =
-   let private testsProjectDirectory = Path.GetFullPath(Paths.Source("Tests"))
-   let private benchmarkOutput = Path.GetFullPath(Paths.Output("benchmarks")) |> directoryInfo
 
-   let private copyToOutput file = CopyFile benchmarkOutput.FullName file
+    let pipelineName = "benchmark-pipeline"
+    let indexName = IndexName.op_Implicit("benchmark-reports")
+    let typeName = TypeName.op_Implicit("benchmarkreport")
+    
+    type Memory(gen0Collections:int, gen1Collections: int, gen2Collections: int, totalOperations:int64, bytesAllocatedPerOperation:int64) =
```

Comment:
While this works and I know we have this types because the json type provider garbled certain types I rather in the long run we don't maintain these POCO's as part of our build script.